variables.tf

```diff
@@ -130,3 +130,9 @@ variable "create_access_log_bucket" {
   default     = false
   description = "A flag to indicate if a bucket for s3 access logs should be created"
 }
+
+variable "allow_ssl_requests_only" {
```

Comment:
What about simply "require_ssl" ?

Reply:
This is the name already used in the underlying s3 bucket module (not my doing) so I kept it consistent